bundles/org.openhab.binding.meater/src/main/java/org/openhab/binding/meater/internal/MeaterAuthenticationException.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.meater.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * {@link MeaterException} is used when there is an authentication exception with MEATER REST API.
```

Comment:
Please fix MeaterAuthenticationException  instead of MeaterException

Reply:
Fixed!